packages/itwin/property-grid/src/hooks/UseDataProvider.ts

```diff
@@ -87,9 +87,11 @@ class PerformanceTrackingProvider implements IPresentationPropertyDataProvider {
     this._wrappedProvider.selectionInfo = selectionInfo;
   }
 
-  public dispose(): void {
-    this._wrappedProvider.dispose();
+  public [Symbol.dispose](): void {
+    this._wrappedProvider[Symbol.dispose]?.();
```

Comment:
I guess this will break if property grid is used with older `presentation-components` versions as older provider won't have `[Symbol.dispose]`?

Reply:
you're right... applied the `safeDispose` pattern we use elsewhere